symengine/visitor.h

```diff
@@ -31,6 +31,7 @@ namespace SymEngine
 class Visitor
 {
 public:
+    virtual ~Visitor(){};
 #define SYMENGINE_ENUM(TypeID, Class) virtual void visit(const Class &) = 0;
```

Comment:
@certik, here are the virtual functions

Reply:
Ah I see, yes. And from those functions we call the bvisit in BaseVisitor.